<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sequenceWindow</class>
 <widget class="QMainWindow" name="sequenceWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>زمان بندی  - تنظیمات</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/time_sequence.png</normaloff>../icons/time_sequence.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>601</width>
      <height>490</height>
     </rect>
    </property>
    <property name="title">
     <string>زمان بندی آپلود</string>
    </property>
    <widget class="QLabel" name="activeClassLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>67</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>دسته ی فعال:</string>
     </property>
    </widget>
    <widget class="ExtendedComboBox" name="activeClassComboBox">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>67</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="newProfilePushButton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>19</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>پروفایل جدید</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteProfilePushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>19</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>حذف پروفایل</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editProfilePushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>19</y>
       <width>76</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ویرایش پروفایل</string>
     </property>
    </widget>
    <widget class="ExtendedComboBox" name="profileComboBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>188</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="profileLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>پروفایل:</string>
     </property>
    </widget>
    <widget class="QLabel" name="startTimeLabel">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>67</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>زمان شروع آپلود:</string>
     </property>
    </widget>
    <widget class="QLabel" name="endTimeLabel">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>97</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>زمان پایان آپلود:</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="startTimeEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>67</y>
       <width>84</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>8</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
     <property name="time">
      <time>
       <hour>8</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QTimeEdit" name="endTimeEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>97</y>
       <width>84</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
     <property name="time">
      <time>
       <hour>17</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSequencePushButton">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>97</y>
       <width>84</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>حذف زمان بندی</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitSequencePushButton">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>67</y>
       <width>84</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ثبت زمانبندی</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>45</y>
       <width>576</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>581</width>
       <height>351</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="videoCountLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>97</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>تعداد ویدئو:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="videoCountSpinBox">
     <property name="geometry">
      <rect>
       <x>404</x>
       <y>97</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateLineEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedComboBox</class>
   <extends>QComboBox</extends>
   <header>source/extended/ExtendedComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
